src/vaadin-dialog-draggable-mixin.html

```diff
@@ -27,8 +27,9 @@
           const isResizerContainer = e.target === resizerContainer;
           const isResizerContainerScrollbar = e.offsetX > resizerContainer.clientWidth || e.offsetY > resizerContainer.clientHeight;
           const isContentPart = e.target === this.$.overlay.$.content;
-          if ((isResizerContainer && !isResizerContainerScrollbar) || isContentPart) {
-            e.preventDefault();
+          const isDraggable = e.target.classList.contains('draggable');
```

Comment:
Using `draggable` as class name, but the name and/or using it as a class or attribute is up for discussion.